docs/src/components/DocNavigation/primitives/Collapse.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { Collapse as OrbitCollapse, mediaQueries } from "@kiwicom/orbit-components";
+import styled, { css } from "styled-components";
+
+const StyledCollapseWrapper = styled.div`
+  > * {
+    border-bottom: 0;
+  }
+`;
+const StyledCollapseLabel = styled.div`
+  font-size: 16px;
+`;
+const StyledCollapseContent = styled.div<{ hasCategories: boolean }>`
+  ${({ hasCategories }) => css`
+    ${hasCategories
+      ? `
+        padding-left: 0.5rem;
+        > * + * {
+          margin-top: 0.5rem;
+        }
+      `
+      : css`
+          ${mediaQueries.desktop(css`
+            padding-left: 0.5rem;
+          `)};
+          > * {
+            padding-left: 0.5rem;
+            padding-right: 0.5rem;
+            ${mediaQueries.desktop(css`
+              padding-left: 0;
+              padding-right: 0;
+            `)};
+          }
+        `}
+  `};
+`;
+
+interface Props extends React.ComponentProps<typeof OrbitCollapse> {
+  hasCategories: boolean;
+}
+
+export default function Collapse({ expanded, label, hasCategories, children, onClick }: Props) {
```

Comment:
it's inside folder `primitives`, in the primitives folder, we usually place only SC primitives, without react components, it's not bad, but it's kinda against our habit, when i open the component inside primitives folder i expect to see only SC components there

so the correct way will be to have separate `Collapse` component and import there these primitives.